src/components/FindMovie/FindMovie.js

```diff
@@ -1,10 +1,33 @@
-import React from 'react';
+import React, {useState, useEffect} from 'react';
+import classNames from 'classnames'
+import {getMovies} from '../../api/api'
 import './FindMovie.scss';
 
 import { MovieCard } from '../MovieCard';
-import movies from '../../api/movies.json';
+//import movies from '../../api/movies.json';
 
-export const FindMovie = () => (
+export const FindMovie = ({onDataRequest}) => {
+  const [errorVisible, setError] = useState(false);
+  const [query, setValue] = useState('');
+  const [newMovie, setNewMovie] = useState();
+  console.log(newMovie)
+
+  const sendRequest = async (movieQuery) => {
+    const movie = await getMovies(movieQuery);
+    const baseUrl = 'https://www.imdb.com/title/'
```

Comment:
```suggestion
    const imdbUrl = 'https://www.imdb.com/title/'
```
It's not a base url, it's just a link to IMDB. You may insert it right into your 'imdbUrl' field